test/features/kogito_service_performance.feature

```diff
@@ -0,0 +1,146 @@
+@performance
+Feature: Kogito Service Performance
+
+  Background:
+    Given Namespace is created
```

Comment:
A new line below would increase readability. Right now background and scenario blocks are hard to distinguish on first look.

Reply:
```suggestion
    Given Namespace is created

```